### PR TITLE
Feature/added components and redux store

### PR DESCRIPTION
**Pull request by angarita-dev**
# Hi :robot: 

This is the pull request for the milestone 2 of the Bookstore project.

### Introduced features:
* Added book reducers.
* Added book actions `createBook` and `removeBook`.
* Added `Book` container.
* Added `BookList` and `BookForm` component.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Provider store={store}>
+      <App />
+    </Provider>
   </React.StrictMode>,
   document.getElementById('root'),
 );
```
Comment (shloch) [inline src/index.js:23]: - You needed to add initial/sample books here [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/InitialBooks.png)]

File: src/containers/Book.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
```
Comment (shloch) [inline src/containers/Book.js]: - this component should be inside the folder `src/components`

File: src/components/BooksList.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
```
Comment (shloch) [inline src/components/BooksList.js]: - relocate component to folder `src/containers`

File: src/components/BooksForm.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
```
Comment (shloch) [inline src/components/BooksForm.js]: - relocate to folder `src/containers`

**Review by shloch — CHANGES_REQUESTED** [review]
## Hello @codingAngarita , 

You really did a good job with coding of the core modules

We however just have small organisational issues, that's all (see below)

Screen Shot 2020-08-18 at 11 30 15 PM

from instructions we have 
- 2 `presentational` components (`App` and `Book`) : they should be in the folder `src/components`
- 02 `container` components (`BooksList` and `BooksForm`) : they should be in the folder `src/containers`

So let's just polish this little detail

Happy coding

**Comment by angarita-dev** [comment]
# Hi! :robot: 
Thanks for the feedback on the organisational problems.

### Introduced changes:
* Moved `BooksForm` and `BooksList` to to the `containers` folder.
* Moved `Book` to the `components` folder.

@codingAngarita :wave:

**Review by bellom — APPROVED** [review]
## Hi @codingAngarita :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**